wordpress-mobile/WordPress-Android
# Feature/add login library tooling

This PR adds a new Fastlane lane that merges the strings from the libraries used by the project into the main strings.xml file. The lane is now configured to look at the login library.
The PR also updates the main strings.xml with the current version of the login library.

Important: this PR disables the `validate-login-library.sh` script on Travis so that developers are not required to add the strings to both libraries during development, _but_ adds the `localize_libs` lane to the `code_freeze` one to ensure that it's executed at every code freeze. 


## To Test:
1. Run `bundle install`
2. Run `bundle exec fastlane localize_libs` and verify that the script is executed without errors and no changes are made.
3. Make some changes to the `strings.xml` file of the Android Login Library, run `bundle exec fastlane localize_libs` again and verify that the changes are merged into the app `strings.xml`.
